core/templates/dev/head/components/entity-creation-services/collection-creation-backend-api.service.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+/**
+ * @fileoverview Service to notify about creation of collection and obtain
+ * collection_id.
+ */
+
+require('domain/utilities/url-interpolation.service.ts');
+require('services/site-analytics.service.ts');
+
+angular.module('oppia').factory('CollectionCreationBackendService', [
+  '$http', '$rootScope', '$timeout', '$window',
+  'SiteAnalyticsService', 'UrlInterpolationService',
+  function(
+      $http, $rootScope, $timeout, $window, SiteAnalyticsService,
+      UrlInterpolationService) {
```

Comment:
Please space the args like lines 23 and 24.

Reply:
The lint check requires these to be spaced in this way, isn't it right?

Reply:
I meant space it like you did for 23 and 24, so it would look like :
```
  function(
      $http, $rootScope, $timeout, $window,
      SiteAnalyticsService, UrlInterpolationService)
```

Reply:
I have made the changes as you have requested, but the lint check is now failing. PTAL.

Reply:
I have made the changes as you have requested, but the lint check is now failing. PTAL.